homeassistant/components/climate/opentherm_gw.py

```diff
@@ -0,0 +1,201 @@
+"""OpenTherm Gateway Climate component for Home Assistant."""
+
+import logging
+import voluptuous as vol
```

Comment:
Add a blank line between standard library and 3rd party imports.

Reply:
addressed in c5a9266871b32a981ad19ab38a0bad650ab644e4